alacritty_terminal/src/term/mod.rs

```diff
@@ -1813,20 +1814,12 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        trace!("Saving cursor position");
-        let cursor = if self.alt { &mut self.cursor_save_alt } else { &mut self.cursor_save };
-
-        *cursor = self.cursor;
+        unimplemented!()
     }
 
     #[inline]
     fn restore_cursor_position(&mut self) {
-        trace!("Restoring cursor position");
-        let source = if self.alt { &self.cursor_save_alt } else { &self.cursor_save };
-
-        self.cursor = *source;
-        self.cursor.point.line = min(self.cursor.point.line, self.grid.num_lines() - 1);
-        self.cursor.point.col = min(self.cursor.point.col, self.grid.num_cols() - 1);
+        unimplemented!()
     }
```

Comment:
I forgot that these are actually called by `ansi.rs`, which makes a lot of sense. We need these functions for things to work properly.

But you can simply call these from `swap_alt`.

Reply:
I looked into it. They seem to be called, but nothing seems to call `csi_dispatch` which is where they're called from. I'll re-implement the methods just in case

Reply:
>but nothing seems to call csi_dispatch

This method is called from outside of alacritty.